packages/core/src/tile/worker/loaders/utils.ts

```diff
@@ -0,0 +1,101 @@
+import { Attribute, Domain } from '../../../types';
+import { Datatype } from '@tiledb-inc/tiledb-cloud/lib/v2';
+import { getQueryDataFromCache } from '../../../utils/cache';
+import { TypedArray } from '../../types';
+
+export function transformBufferToInt64(
+  buffer: ArrayBuffer,
+  attribute: Attribute
+): BigInt64Array {
+  switch (attribute.type) {
+    case Datatype.Int32:
+      return BigInt64Array.from(
```

Comment:
Is there any reason we convert `Int32` to bigInt?

Reply:
I need all ids to be int64 because that's how the renderer is setup to support picking